src/SWP/Bundle/StorageBundle/Doctrine/ORM/EntityRepositoryTrait.php

```diff
@@ -149,7 +149,12 @@ protected function applyCriteria(QueryBuilder $queryBuilder, Criteria $criteria,
      */
     protected function applySorting(QueryBuilder $queryBuilder, array $sorting, string $alias, Criteria $criteria = null)
     {
+        $properties = array_merge($this->getClassMetadata()->getFieldNames(), $this->getClassMetadata()->getAssociationNames());
         foreach ($sorting as $property => $order) {
+            if (!in_array($property, $properties)) {
```

Comment:
`Using '\in_array(...)' would enable some of opcode optimizations` and `Third parameter should be provided to clarify if type safety is important in this context`